src/java/org/apache/cassandra/service/reads/repair/BlockingReadRepairs.java

```diff
@@ -33,14 +33,14 @@
 import org.apache.cassandra.schema.TableMetadata;
 import org.apache.cassandra.tracing.Tracing;
 
+import static org.apache.cassandra.config.CassandraRelevantProperties.DROP_OVERSIZED_READREPAIR_MUTATIONS;
 import static org.apache.cassandra.db.IMutation.MAX_MUTATION_SIZE;
 
 public class BlockingReadRepairs
 {
     private static final Logger logger = LoggerFactory.getLogger(BlockingReadRepairs.class);
 
-    private static final boolean DROP_OVERSIZED_READ_REPAIR_MUTATIONS =
-        Boolean.getBoolean("cassandra.drop_oversized_readrepair_mutations");
+    private static final boolean DROP_OVERSIZED_READ_REPAIR_MUTATIONS = DROP_OVERSIZED_READREPAIR_MUTATIONS.getBoolean();
```

Comment:
That is quite unfortunate that these names differ only in one underscore.

Reply:
Fixed.